data/json/itemgroups/SUS/alien.json

```diff
@@ -57,12 +57,12 @@
     "entries": [
       { "group": "SUS_exodii_storage_lowvalue", "prob": 30 },
       {
-        "collection": [ { "item": "pamd68", "count": [ 15, 20 ] }, { "item": "pamd71z", "count": [ 10, 15 ] } ],
```

Comment:
These Exodii storage crates were spawning thousands and thousands of pamd variants all over the overmap.  I drastically dropped counts here, and I decreased the spawns of the locations that spawn them.  It shouldn't be meaningfully harder to find the main faction location though.

Reply:
Weren't these locations the main reliable place to find cyberware after most of the endgame spawns were removed from the labs?